### PR TITLE
Print `LazyTensorOperation` in the form "%id = op[...](...)".

### PR DESCRIPTION
**Pull request by bgogul**

### CODE REVIEW

File: Sources/TensorFlow/LazyTensor/LazyTensorOperation.swift [hunk 1]
```diff
@@ -147,6 +147,34 @@ class LazyTensorOperation: TensorOperation {
         }
     }
 
+    func outputName(_ index: Int) -> String {
```
Comment (rxwei) [inline Sources/TensorFlow/LazyTensor/LazyTensorOperation.swift]:
```suggestion
    func outputName(at index: Int) -> String {
```

Comment (bgogul) [reply Sources/TensorFlow/LazyTensor/LazyTensorOperation.swift]: Done.

File: Sources/TensorFlow/LazyTensor/LazyTensorOperation.swift [hunk 2]
```diff
@@ -147,6 +147,36 @@ class LazyTensorOperation: TensorOperation {
...
+        }
+        return ssaName
+    }
+
+    var outputName: String {
+        switch outputCount {
+        case 0: return ""
+        case 1: return "\(outputName(at: 0))"
+        default:
+            do {
```
Comment (rxwei) [inline Sources/TensorFlow/LazyTensor/LazyTensorOperation.swift]: Why use a `do` block here?

File: Sources/TensorFlow/LazyTensor/LazyTensorOperation.swift [hunk 3]
```diff
@@ -147,6 +147,36 @@ class LazyTensorOperation: TensorOperation {
         }
     }
 
+    func outputName(at index: Int) -> String {
+        precondition(index < outputCount,
+            "Output index out of bounds when getting outputName.")
+        var ssaName = ""
+        if let id = self.id {
```
Comment (rxwei) [inline Sources/TensorFlow/LazyTensor/LazyTensorOperation.swift]:
```suggestion
        ssaName = "%\(id ?? ObjectIdentifier(self))"
```

File: Sources/TensorFlow/LazyTensor/LazyTensorOperation.swift [hunk 4]
```diff
@@ -147,6 +147,36 @@ class LazyTensorOperation: TensorOperation {
...
+        return ssaName
+    }
+
+    var outputName: String {
+        switch outputCount {
+        case 0: return ""
+        case 1: return "\(outputName(at: 0))"
+        default:
+            do {
+                let outputNames = (0..<outputCount).map {
```
Comment (rxwei) [inline Sources/TensorFlow/LazyTensor/LazyTensorOperation.swift]: Use `.lazy` so it won't create an array first.

```suggestion
                let outputNames = (0..<outputCount).lazy.map {
```

File: Sources/TensorFlow/LazyTensor/LazyTensorOperation.swift [hunk 5]
```diff
@@ -630,8 +660,8 @@ extension LazyTensor: CustomStringConvertible {
                 : "\(h.valueDescription)"
         case LazyTensor.Handle.symbolic(let op, let index, let isLive):
             return isLive
```
Comment (rxwei) [inline Sources/TensorFlow/LazyTensor/LazyTensorOperation.swift]:
```suggestion
            return op.outputName(at: index) + isLive ? "*" : ""
```

File: Sources/TensorFlow/LazyTensor/LazyTensorOperation.swift (hunk 2 above)
Comment (bgogul) [reply Sources/TensorFlow/LazyTensor/LazyTensorOperation.swift]: ah, habit from C++ to put `case` blocks within {}. Removed it.

File: Sources/TensorFlow/LazyTensor/LazyTensorOperation.swift [hunk 6]
```diff
@@ -147,6 +147,36 @@ class LazyTensorOperation: TensorOperation {
...
+        if outputCount > 1 {
+            ssaName += ".\(index)"
+        }
+        return ssaName
+    }
+
+    var outputName: String {
+        switch outputCount {
+        case 0: return ""
+        case 1: return "\(outputName(at: 0))"
```
Comment (rxwei) [inline Sources/TensorFlow/LazyTensor/LazyTensorOperation.swift]:
```suggestion
        case 1: return outputName(at: 0)
```

Comment (bgogul) [reply Sources/TensorFlow/LazyTensor/LazyTensorOperation.swift]: Ah, done.

File: Sources/TensorFlow/LazyTensor/LazyTensorOperation.swift (hunk 4 above)
Comment (bgogul) [reply Sources/TensorFlow/LazyTensor/LazyTensorOperation.swift]: Nice.. Thanks! Done.

File: Sources/TensorFlow/LazyTensor/LazyTensorOperation.swift (hunk 3 above)
Comment (bgogul) [reply Sources/TensorFlow/LazyTensor/LazyTensorOperation.swift]: done

**Pull request merged**